nadeo_event_api/src/nadeo_event_api/api/structure/settings/script_settings.py

```diff
@@ -2,17 +2,42 @@
 
 from ...structure.enums import RespawnBehavior
 
+# Nadeo api documentation: https://wiki.trackmania.io/en/dedicated-server/Usage/OfficialGameModesSettings
 
-class ScriptSettings(ABC):
+# 
+class BaseScriptSettings():
     def __init__(
-        self,
+        self,         
```

Comment:
Run black formatter `black .` to remove these empty spaces and overall formatting issues